scribejava-core/src/main/java/com/github/scribejava/core/oauth/OAuth20Service.java

```diff
@@ -172,6 +172,8 @@ public final OAuth2AccessToken getAccessTokenPasswordGrant(String uname, String
 
         request.addParameter(OAuthConstants.GRANT_TYPE, OAuthConstants.PASSWORD);
 
+        request.addParameter(OAuthConstants.CLIENT_ID, config.getApiKey());
+        request.addParameter(OAuthConstants.CLIENT_SECRET, config.getApiSecret());
```

Comment:
according to https://tools.ietf.org/html/rfc6749#section-4.3.1
this data is not required to send.
Maybe it's better to make them optional?
Like this
```
if(config.hasClientCredentials()) {
...
}
```